beacon-chain/core/state/transition.go

```diff
@@ -232,7 +231,7 @@ func ProcessOperations(
 	ctx context.Context,
 	state *pb.BeaconState,
 	body *ethpb.BeaconBlockBody,
-	config *TransitionConfig) (*pb.BeaconState, error) {
```

Comment:
Please delete the entire struct TransitionConfig

Reply:
I would but verifyStateRoot still requires it

Reply:
Discussed offline that this will be resolved in @terencechain upcoming PR.